packages/components/spacings/spacings-inset/src/inset.tsx

```diff
@@ -24,22 +24,24 @@ const getPadding = (scale?: TScale) => {
 
 export type TInsetProps = {
   scale: TScale;
+  height: 'collapsed' | 'expanded';
```

Comment:
Nit: sorry to be picky but I feel like the naming is a bit confusing 🤔

On one hand we don't accept any "normal" CSS values for `height` but the prop is named `height`.
On the other hand, it seems that `collapsed` is not really used and the prop is meant to be more of a boolean.

Could you clarify these points and the reasoning behind the naming?

Reply:
Please make a counter suggestion as a point for discussion. I feel this is the best current naming we have. The prior boolean prop was abandoned to avoid "boolean prop hell" and to have an enum for scalability.

Reply:
I'm just trying to understand the background 🙂

On the top of my head I can think of `heightMode` or something similar.

Reply:
What about `verticalSpacing` or `verticalSpace` with `collapse` and `expand`? I like `height` more than `*Mode` but a careful of the naming rabbit whole. @CarlosCortizasCT you can choose. I am happy to make an adjustment so you don't have to again 🤗

Reply:
What if we keep the `height` property but we use known CSS values?
* [max-content](https://developer.mozilla.org/en-US/docs/Web/CSS/height#max-content): The intrinsic preferred height.
* [min-content](https://developer.mozilla.org/en-US/docs/Web/CSS/height#min-content): The intrinsic minimum height.

Reply:
Good idea. I like that. ➕➕

Reply:
Updated here: c0256a1628f3319d03fd9a5d59307b7b62f0042e